test/ccall.jl

```diff
@@ -712,4 +777,68 @@ elseif Sys.ARCH === :aarch64
+                         (5.0, 6.0, 7.0, 8.0),
+                         (11.0, 12.0, 13.0, 14.0),
+                         (15.0, 16.0, 17.0, 18.0),
+                         (21.0, 22.0, 23.0, 24.0),
+                         (25.0, 26.0, 27.0, 28.0),
+                         (31.0, 32.0, 33.0, 34.0),
+                         (35.0, 36.0, 37.0, 38.0),
+                         (41.0, 42.0, 43.0, 44.0)),))
+@test_huge 6 "_ppc64" ((((1.0, 2.0, 3.0, 4.0),
+                         (5.0, 6.0, 7.0, 8.0),
+                         (11.0, 12.0, 13.0, 14.0),
+                         (15.0, 16.0, 17.0, 18.0),
+                         (21.0, 22.0, 23.0, 24.0),
+                         (25.0, 26.0, 27.0, 28.0),
+                         (31.0, 32.0, 33.0, 34.0),
+                         (35.0, 36.0, 37.0, 38.0)),
+                        (41.0, 42.0, 43.0, 44.0)))
+@test_huge 1 "_ppc64_hva" ((((1.0, 2.0, 3.0, 4.0),
+                             (5.0, 6.0, 7.0, 8.0),
+                             (11.0, 12.0, 13.0, 14.0),
+                             (15.0, 16.0, 17.0, 18.0),
+                             (21.0, 22.0, 23.0, 24.0),
+                             (25.0, 26.0, 27.0, 28.0),
+                             (31.0, 32.0, 33.0, 34.0),
+                             (35.0, 36.0, 37.0, 38.0)),))
+@test_huge 2 "_ppc64_hva" (((((1.0, 2.0, 3.0, 4.0),
+                              (5.0, 6.0, 7.0, 8.0)),
+                             ((11.0, 12.0, 13.0, 14.0),
+                              (15.0, 16.0, 17.0, 18.0))),))
+@test_huge 3 "_ppc64_hva" (((1.0, 2.0, 3.0, 4.0),
+                            (((11.0, 12.0, 13.0, 14.0),
+                              (15.0, 16.0, 17.0, 18.0)),)))
+@test_huge 4 "_ppc64_hva" (((1, 2, 3, 4),
+                            (11.0, 12.0, 13.0, 14.0)))
+@test_huge 5 "_ppc64_hva" (((1, 2, 3, 4),
+                            (11.0, 12.0)))
+
+@test 18451 == ccall((:test_ppc64_vec1long, libccalltest), Int64,
+    (Int64, Int64, Int64, Int64, Int64, Int64, Int64, Int64, Int64, Struct_huge1_ppc64),
+    1, 2, 3, 4, 5, 6, 7, 8, 9, Struct_huge1_ppc64(18000, (100, 101, 102, 103)))
+
+@test 941 == ccall((:test_ppc64_vec1long_vec, libccalltest), Int64,
+    (Int64, Int64, Int64, Int64, Int64, Int64, Int64, Int64, Int64, V4xF32),
+    11, 12, 13, 14, 15, 16, 17, 18, 19, (200, 201, 202, 203))
+
+@test V4xF32((614232, 614218, 614204, 614190)) ==
+     ccall((:test_ppc64_vec2, libccalltest), V4xF32,
+    (Int64, V4xF32, V4xF32, V4xF32, V4xF32,
+     V4xF32, V4xF32, V4xF32, V4xF32, V4xF32,
+     V4xF32, V4xF32, V4xF32, V4xF32, V4xF32),
+    600000, (4, 3, 2, 1), (5, 4, 3, 2), (6, 5, 4, 3), (7, 6, 5, 4),
+    (14, 13, 12, 11), (15, 14, 13, 12), (16, 15, 14, 13), (17, 16, 15, 14), (18, 17, 16, 15),
+    (1024, 1023, 1022, 1021), (1025, 1024, 1023, 1022), (1026, 1025, 1024, 1023), (1027, 1026, 1025, 1024), (10028, 10027, 10026, 10025))
+
+else
+warn("ccall: no VecReg tests run for this platform")
```

Comment:
we should be running VecReg tests on all supported platforms, or we are missing test coverage